system-int-test/test/cmr/system_int_test/search/granule_orbit_search_test.clj

```diff
@@ -9,7 +9,7 @@
    [cmr.spatial.kml :as kml]
    [cmr.spatial.line-string :as l]
    [cmr.spatial.mbr :as m]
-   [cmr.spatial.point :as p]
+   [cmr.spatial.point :as point]
```

Comment:
👏

Reply:
boo! Now `point` feels lonely because he is forced to move away from his friends `l` and `m`. :)